views/pages/gallery.ejs

```diff
@@ -7,18 +7,30 @@
 <main>
   <h1>GALLERY</h1>
 
-  <%= console.log('FRONT END ARR: ', dataArray) %>
-
-  <div class="resultContainer">
-    <% for (data of dataArray) { %>
-      <div class="resultBox">
-        <img src="<%= data.img %>" alt="">
-        <h3><%= data.title %></h3>
-        <p><%= data.artist %></p>
-      </div>
-    <% } %>
-  </div>
-  <!-- ADD ALL THE THINGS -->
+  <!-- turn this into a partial? -->
+  <form action="/gallery"><input type="submit" value="Get Inspired!"></form>
+
+  <section class="gallery-card">
+    <div class="resultContainer">
```

Comment:
we will ultimately end up using the same class name on both sections for favorites and gallery but this is good for now, we can streamline stuff tomorrow :D